db/seeds.rb

```diff
@@ -6,24 +6,30 @@
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
 Article.delete_all
+Category.delete_all
 
-articles = [{ 'title' => Faker::Book.title, 'body' => Faker::Quote.matz, 'category' => 'news' },
-            { 'title' => Faker::Book.title, 'body' => Faker::Quote.matz, 'category' => 'news'  },
-            { 'title' => Faker::Book.title, 'body' => Faker::Quote.matz, 'category' => 'news'  },
-            { 'title' => Faker::Book.title, 'body' => Faker::Quote.matz, 'category' => 'news'  },
-            { 'title' => Faker::Book.title, 'body' => Faker::Quote.matz, 'category' => 'news'  },
-            { 'title' => Faker::Book.title, 'body' => Faker::Quote.matz, 'category' => 'politics'  },
-            { 'title' => Faker::Book.title, 'body' => Faker::Quote.matz, 'category' => 'politics'  },
-            { 'title' => Faker::Book.title, 'body' => Faker::Quote.matz, 'category' => 'politics'  },
-            { 'title' => Faker::Book.title, 'body' => Faker::Quote.matz, 'category' => 'politics'  },
-            { 'title' => Faker::Book.title, 'body' => Faker::Quote.matz, 'category' => 'sports'  },
-            { 'title' => Faker::Book.title, 'body' => Faker::Quote.matz, 'category' => 'sports'  },
-            { 'title' => Faker::Book.title, 'body' => Faker::Quote.matz, 'category' => 'sports'  },
-            { 'title' => Faker::Book.title, 'body' => Faker::Quote.matz, 'category' => 'sports'  },
-            { 'title' => Faker::Book.title, 'body' => Faker::Quote.matz, 'category' => 'economy'  },
-            { 'title' => Faker::Book.title, 'body' => Faker::Quote.matz, 'category' => 'economy'  },
-            { 'title' => Faker::Book.title, 'body' => Faker::Quote.matz, 'category' => 'economy'  }]
+category_sports = Category.create(name: 'sports')
+category_politics = Category.create(name: 'politics')
+category_economy = Category.create(name: 'economy')
+category_news = Category.create(name: 'news')
+
+articles = [{ title: Faker::Book.title, body: Faker::Quote.matz, category_id: category_sports.id },
```

Comment:
;-) I don't remember how many times I repeated the in ActiveRecord you don't have to pass in the id but rather the entire object. 

```ruby
title: Faker::Book.title, body: Faker::Quote.matz, category: category_sports }
```